### PR TITLE
Feature/change user info

### PR DESCRIPTION
**Pull request by SolovyovDmitry**
https://trello.com/c/Qu4RFCMP/144-filterservice-remove-avatars

### CODE REVIEW

File: src/FilterService.Mappers/Models/UserInfoMapper.cs [hunk 1]
```diff
@@ -28,43 +28,22 @@ public class UserInfoMapper : IUserInfoMapper
...
+      return usersData?.Select(x => new UserInfo
       {
         Id = x.Id,
+        AvatarId = x.ImageId,
         FirstName = x.FirstName,
         LastName = x.LastName,
         MiddleName = x?.MiddleName,
         Position = positionInfo?.FirstOrDefault(pi => pi.Id == positionFilteredData?.FirstOrDefault(pfd => pfd.UsersIds.Contains(x.Id))?.Id) ??
-          positionInfo?.FirstOrDefault(pi => pi.Id == positionData?.FirstOrDefault(pd => pd.UsersIds.Contains(x.Id))?.Id),
+                   positionInfo?.FirstOrDefault(pi => pi.Id == positionData?.FirstOrDefault(pd => pd.UsersIds.Contains(x.Id))?.Id),
```
Comment (VLiamin) [inline src/FilterService.Mappers/Models/UserInfoMapper.cs:39]: tabs?

**Review by VLiamin — APPROVED** [review]

File: src/FilterService.Mappers/Models/UserInfoMapper.cs [hunk 2]
```diff
@@ -28,14 +28,10 @@ public class UserInfoMapper : IUserInfoMapper
...
-      if (usersData is null)
-      {
-        return null;
-      }
-
-      return usersData.Select(x => new UserInfo
+      return usersData?.Select(x => new UserInfo
       {
         Id = x.Id,
+        AvatarId = x.ImageId,
```
Comment (vorobevamv) [inline src/FilterService.Mappers/Models/UserInfoMapper.cs]: AvatarId = x?.ImageId
and please, move to line 38, after MiddleName

**Review by vorobevamv — APPROVED** [review]

**Comment by vorobevamv** [comment]
remove IImageInfoMapper+ImageInfoMapper, IImageService+ImageService, ImageInfo (from Models.Dto);
in appsettings remove
//ImageService
    "GetImagesEndpoint": "ImageService_GetImages"

in StartUp update version;
in Command remove unnecessary private variables + clean usings

File: src/FilterService.Mappers/Models/UserInfoMapper.cs (hunk 2 above)
Comment (SolovyovDmitry) [reply src/FilterService.Mappers/Models/UserInfoMapper.cs]: userData is known to be not null

**Pull request merged**